src/coreclr/tools/StressLogAnalyzer/StressLogPlugin.cpp

```diff
@@ -779,10 +779,7 @@ bool ParseOptions(int argc, char* argv[])
             case 'O':
                 if (arg[2] == ':')
                 {
-                    WCHAR* buffer = new WCHAR[1000];
-                    if (MultiByteToWideChar(CP_ACP, 0, &arg[3], -1, buffer, 1000) == 0)
```

Comment:
Just noticed a similar method `convertC` which is unused in `src/mono/dlls/dbgshim/dbgshim.cpp` (can be deleted).